### ISSUE TITLE
Translations update

### ISSUE THREAD

**Issue by nickik** [redox-os/website#173]
The new community.<lang>.md files need to be translated.

Also the titles in config.toml.

**Comment by wright08** [comment]
I dont mind taking a stab at trying to run the files through Google Translate. Would this be a sufficient fix to the problem?

**Comment by xTibor** [comment]
Nope, I wouldn't do this. Not even as a last resort. Machine translation, especially Google's *throw a deep learning algorithm at the problem and see what happens* method produces horrible results. Their translations are inaccurate, unnatural and unintelligible for slightly complex texts.

Just look at [this example](https://i.imgur.com/OE74cIe.png) from a news website I ran through their translator. This is the quality you'd be dealing with. It's so inaccurate it wasn't even able to translate the title of that article properly ("Dogs torn woman into pieces").

**Comment by wright08** [comment]
I understand Google Translate isn't perfect, but at least having some of it translated could be beneficial.

Maybe we translate it with Google in the meantime and make a note that the translations were automatically generated and show users where they could look to help if they are so inclined.
